### PR TITLE
chore: backport various CVE bumps to 6.0

### PR DESCRIPTION
**Pull request by agavra**
### Description 

see https://github.com/confluentinc/ksql/pull/7493

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -130,6 +130,14 @@
...
         <git-commit-id-plugin.version>2.2.6</git-commit-id-plugin.version>
         <scala.version>2.13.2</scala.version>
+
+        <netty-tcnative-version>2.0.39.Final</netty-tcnative-version>
+        <!-- We normally get this from common, but Vertx is built against 4.1.49.Final -->
+        <netty.version>4.1.60.Final</netty.version>
+        <!-- This matches the version in common -->
+        <netty-codec-http2-version>4.1.49.Final</netty-codec-http2-version>
+        <!-- Brought in by both schema registry and connect, we need to pick a version -->
+        <jersey-common>2.31</jersey-common>
```
Comment (xjin-Confluent) [inline pom.xml:140]: The fixed version is 2.34 though

Comment (agavra) [reply pom.xml:140]: thanks for catching this, not sure where my copy-pasta went wrong. looks like the above is also not right (hsoudl be .60.Final)

Comment (xjin-Confluent) [reply pom.xml:140]: Yep https://github.com/confluentinc/ksql/blob/6.2.x/pom.xml#L144
I don't see jersey defined in 6.2.x though.

Comment (agavra) [reply pom.xml:140]: I took it from https://github.com/confluentinc/ksql/commit/2425f163134a823871d6756c23d39e795f117d6c

**Comment by xjin-Confluent** [comment]
Blocked by https://github.com/confluentinc/kafka/pull/574

**Comment by agavra** [comment]
For context on why it's blocked by the above:
```
[INFO] --- maven-dependency-plugin:3.1.2:tree (default-cli) @ ksqldb-rest-app ---
[INFO] io.confluent.ksql:ksqldb-rest-app:jar:6.0.3-SNAPSHOT
[INFO] \- io.confluent.ksql:ksqldb-rest-model:jar:6.0.3-SNAPSHOT:compile
[INFO]    \- org.apache.kafka:connect-runtime:jar:6.0.3-ccs-SNAPSHOT:compile
[INFO]       \- org.glassfish.jersey.containers:jersey-container-servlet:jar:2.31:compile
[INFO]          +- org.glassfish.jersey.core:jersey-common:jar:2.31:compile
[INFO]          \- org.glassfish.jersey.core:jersey-server:jar:2.31:compile
[INFO]             \- org.glassfish.jersey.core:jersey-client:jar:2.31:compile
```

**Review by xjin-Confluent — APPROVED** [review]

**Comment by xjin-Confluent** [comment]
@agavra  Is this pint merged to 6.1.x of ksql?

**Comment by agavra** [comment]
@xjin-Confluent yes

**Pull request merged**